libmalcontent/manager.c

```diff
@@ -434,6 +435,29 @@ mct_manager_get_app_filter (MctManager            *self,
       return NULL;
     }
 
+  result_variant2 =
+      g_dbus_connection_call_sync (self->connection,
+                                   "org.freedesktop.Accounts",
+                                   object_path,
+                                   "org.freedesktop.DBus.Properties",
+                                   "GetAll",
+                                   g_variant_new ("(s)", "org.freedesktop.Accounts.User"),
+                                   G_VARIANT_TYPE ("(a{sv})"),
+                                   (flags & MCT_GET_APP_FILTER_FLAGS_INTERACTIVE)
+                                     ? G_DBUS_CALL_FLAGS_ALLOW_INTERACTIVE_AUTHORIZATION
+                                     : G_DBUS_CALL_FLAGS_NONE,
+                                   -1,  /* timeout, ms */
+                                   cancellable,
+                                   &local_error);
```

Comment:
Changes lgtm to me but I wonder if we could just get the AccountType property from the user object instead of all properties?

Reply:
Doing it this way is not particularly slower, and meant I could copy/paste the existing code. I can redo it if you want.

Reply:
Right, not worth it given this is a temporary change anyway.